<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainDialog</class>
 <widget class="QDialog" name="MainDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>450</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../maindialog.qrc">
    <normaloff>:/images/images/python-icon.svg</normaloff>:/images/images/python-icon.svg</iconset>
  </property>
  <layout class="QVBoxLayout" name="vl_main_dialog">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <widget class="QGroupBox" name="gb_input">
     <property name="title">
      <string>Input</string>
     </property>
     <layout class="QHBoxLayout" name="hl_gb_input_main">
      <property name="leftMargin">
       <number>3</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>3</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="hl_gb_input">
        <property name="leftMargin">
         <number>3</number>
        </property>
        <property name="topMargin">
         <number>3</number>
        </property>
        <property name="rightMargin">
         <number>3</number>
        </property>
        <property name="bottomMargin">
         <number>3</number>
        </property>
        <item>
         <widget class="QPlainTextEdit" name="pte_input_text">
          <property name="acceptDrops">
           <bool>false</bool>
          </property>
          <property name="lineWrapMode">
           <enum>QPlainTextEdit::NoWrap</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="vl_input_buttons">
          <item>
           <widget class="QPushButton" name="pb_copy_from_clipb">
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Copy information on the clipboard to the input area.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="text">
             <string>Paste from Clipboard</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="hl_imp_enc">
            <property name="leftMargin">
             <number>5</number>
            </property>
            <item>
             <widget class="QLabel" name="lb_imp_enc">
              <property name="text">
               <string>Enc:</string>
              </property>
              <property name="buddy">
               <cstring>cb_imp_enc</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="cb_imp_enc"/>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="pb_import_from_file">
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Import the content from a file into the input area (file encoding can be selected).&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="text">
             <string>Import from File</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="vs_input_buttons">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pb_reset">
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Remove the content of in- and output area and reset the options.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="text">
             <string>Reset</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pb_convert">
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Perform the conversion from input to output.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="text">
             <string>Convert</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gb_output">
     <property name="title">
      <string>Output</string>
     </property>
     <layout class="QHBoxLayout" name="hl_gb_output_main">
      <property name="leftMargin">
       <number>3</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>3</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="hl_gb_output">
        <property name="leftMargin">
         <number>3</number>
        </property>
        <property name="topMargin">
         <number>3</number>
        </property>
        <property name="rightMargin">
         <number>3</number>
        </property>
        <property name="bottomMargin">
         <number>3</number>
        </property>
        <item>
         <widget class="QPlainTextEdit" name="pte_output_text">
          <property name="acceptDrops">
           <bool>false</bool>
          </property>
          <property name="lineWrapMode">
           <enum>QPlainTextEdit::NoWrap</enum>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="vl_output_buttons">
          <item>
           <widget class="QPushButton" name="pb_copy_to_clipb">
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Copy the output area as text to the clipboard.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="text">
             <string>Copy to Clipboard</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="hl_exp_enc">
            <property name="leftMargin">
             <number>5</number>
            </property>
            <item>
             <widget class="QLabel" name="lb_exp_enc">
              <property name="text">
               <string>Enc.:</string>
              </property>
              <property name="buddy">
               <cstring>cb_exp_enc</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="cb_exp_enc"/>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="pb_export_to_file">
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Export the content of the output area to a file (file encoding can be selected).&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="text">
             <string>Export to File</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="vs_output_buttons">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pb_about">
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Get version information.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="text">
             <string>About</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pb_exit">
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Leave program.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="text">
             <string>Exit</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>pte_input_text</tabstop>
  <tabstop>pb_copy_from_clipb</tabstop>
  <tabstop>cb_imp_enc</tabstop>
  <tabstop>pb_import_from_file</tabstop>
  <tabstop>pb_reset</tabstop>
  <tabstop>pb_convert</tabstop>
  <tabstop>pb_copy_to_clipb</tabstop>
  <tabstop>cb_exp_enc</tabstop>
  <tabstop>pb_export_to_file</tabstop>
  <tabstop>pb_about</tabstop>
  <tabstop>pb_exit</tabstop>
  <tabstop>pte_output_text</tabstop>
 </tabstops>
 <resources>
  <include location="../maindialog.qrc"/>
 </resources>
 <connections/>
</ui>
